build/update-readme.ts

```diff
@@ -0,0 +1,14 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import { readFileSync, writeFileSync } from 'fs-extra';
+
+const readme = readFileSync('./README.md');
+
+const lines = `${readme}`.split('\n');
+
+const index = lines.findIndex((line) => line.includes('## Overview'));
```

Comment:
The README doesn't contain the heading "Overview", so this does nothing. I think it's worth removing this build script, as well as line 60 in the GitHub Action which calls this script.

Reply:
In fact `node ./out/build/update-readme.js` fails for me with `Error: Cannot find module '/home/runner/work/vscode-yaml/vscode-yaml/out/build/update-readme.js'`